mail_notification_custom_subject/readme/CONFIGURE.rst

```diff
@@ -0,0 +1,8 @@
+
+* Activate access to **Technical Features** (debug mode).
+* Go to **Settings > Technical > Email > Subject Replacement Templates**
+* Create a new template.
+
+    * The field **Model** specifies the model to which the subject template should apply in the notification emails sent by Odoo.
+    * The field **Subject Template** accepts `Jinja <https://jinja.palletsprojects.com/en/2.11.x/>`__ expressions.
```

Comment:
Does this use normal Jinja2 syntax, or Odoo-specific syntax? I think it would be good to specify it. Odoo syntax has some limitations, as you see in https://github.com/odoo/odoo/issues/45611 and https://github.com/odoo/odoo/pull/67868.

Reply:
It's not the same. This is one line and raw text, not HTML, so no possible problems like the ones you point. It's pure Jinja.